modules/ROOT/pages/policy-visualization.adoc

```diff
@@ -0,0 +1,51 @@
+= Policy Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== About the Policy Visualization
+
+[NOTE]
+When deploying a Mule app, you must ensure that you are using a version of Mule that supports the Policy Visualization. Learn more about which xref:setup.adoc#select-a-deployment-type[runtimes are supported].
```

Comment:
Should this info begin the other visualization topics? Regardless, I suggest...:
```suggestion
To benefit from the Policy Visualization, you must be using a version of Mule runtime engine (Mule) that supports it.
```
Then move the link to the bottom.

Reply:
I'm not sure we want to go "back" to deployment at this point in the doc.